libs/logging/src/log_source.ts

```diff
@@ -0,0 +1,24 @@
+export enum LogSource {
+  System = 'system',
+  VxAdminApp = 'vx-admin',
+  VxAdminServer = 'vx-admin-server',
+  VxBatchScanApp = 'vx-batch-scan',
+  VxBatchScanSever = 'vx-batch-scan-server',
```

Comment:
I see that this was just moved rather than added in this commit, but I wonder if it should be using this?
```suggestion
  VxBatchScanFrontend = 'vx-batch-scan-frontend',
  VxBatchScanService = 'vx-batch-scan-service',
```